flatgfa-py/src/lib.rs

```diff
@@ -444,8 +470,131 @@ impl PyPath {
         self.steps().__len__()
     }
 }
+#[derive(Clone)]
+#[pyclass(frozen)]
+#[pyo3(name = "ChunkEvent", module = "flatgfa")]
+struct PyChunkEvent {
+    chunk_event: Arc<ChunkEvent>,
```

Comment:
Because the `ChunkEvent` struct is "just plain data," I think it makes a little more sense to just copy it instead of of using an `Arc` to refer to it. That is, we can add `#derive(Clone)` to `ChunkEvent` and then store a copy here (of type `ChunkEvent`, instead of `Arc<ChunkEvent>`).